src/Symfony/Component/Process/Process.php

```diff
@@ -358,6 +358,11 @@ public function start(?callable $callback = null, array $env = []): void
 
         try {
             $process = @proc_open($commandline, $descriptors, $this->processPipes->pipes, $this->cwd, $envPairs, $this->options);
+
+            // Backwards compatibility. To be removed in version 8.0
```

Comment:
I wouldn't change the behavior in 8.0, it's fins e as is.
```suggestion
            // Ensure array vs string commands behave the same
```